root/etc/apparmor.d/docker_fuse

```diff
@@ -9,6 +12,8 @@ profile docker_fuse flags=(attach_disconnected,mediate_deleted) {
   umount,
   mount,
 
+  # Allow signals send to unconfined, usually systemd-shutdown ends up sending these out
+  signal (send) peer="unconfined",
```

Comment:
isn't it received signals from systemd-unconfined?

Reply:
No, it sure looked like it was send from the audit log. I'll double check.

Reply:
```Mar 24 22:48:19 titusagent-devint01cell001-r5metal000-i-0b13e629e19cb6726 kernel: [106147.137358] audit: type=1400 audit(1616626099.646:222): apparmor="DENIED" operation="signal" profile="docker_titus" pid=2006590 comm="systemd-shutdow" requested_mask="send" denied_mask="send" signal=cont peer="docker_titus//sshd"```

Reply:
Oh I'm wrong about the peer, I could make it be sshd.

So this is 

    systemd-shutdown in the container -> signaling titus-sshd to die

Reply:
fixed